src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.Utils;
+
+public record Cars(List<Car> carList) {
+    public static final String CAR_NAME_DELIMITER = ",";
+
+    public static Cars createByString(String carsString) {
+        List<Car> carList = Utils.splitByDelimiter(carsString, CAR_NAME_DELIMITER)
+                .stream()
+                .map(CarName::new)
+                .map(Car::new)
+                .toList();
+        return new Cars(carList);
+    }
+
+    public void commandAllCarsToMove() {
+        carList.forEach(car ->
+                car.moveForwardIfTrue(Utils.isTrueWithProbability60Percent())
+        );
+    }
+
+    public List<Car> getWinnerList() {
+        Integer winnerPosition = getWinnerPosition();
+        return carList.stream()
+                .filter(car -> car.isSamePosition(winnerPosition))
+                .collect(Collectors.toList());
```

Comment:
java 17 문법에 의해, 이 부분을 간결하게 정리 가능합니다.
```suggestion
                .toList();
```

Reply:
@h-beeen 
아하! JDK17에서는 저렇게 단순하게 바꿀 수 있군요! 지적해주셔서 감사합니다!👍

Reply:
저도 이부분에 대해서 찾아봤었는데 참고했던 블로그 같이 남깁니다!
https://binux.tistory.com/146

Reply:
@leemiyeonn 
와,, 둘이 같은 역할을 하는 줄 알았는데 전혀 아니였군요! 감사해요!!! 새로운 것을 배워갑니다!😁